main.star

```diff
@@ -276,6 +269,13 @@ def run(plan, args):
         ),
     )
 
+    # Start default permissionless node.
+    # Note that an additional suffix will be added to the services.
+    permissionless_args = args
+    permissionless_args["deployment_suffix"] = "-pless" + args["deployment_suffix"]
+    permissionless_args["genesis_artifact"] = genesis_artifact
```

Comment:
The thing with this is that args will get copied, but there is an aliasing issue. The `deployment_suffix` and `genesis_artifact` will also be in `args`.

Reply:
Good catch, I used `dict` to create a shallow copy instead in [2b929a3](https://github.com/0xPolygon/kurtosis-cdk/pull/22/commits/2b929a3382044a7d6e0b04e80094a0f4a4614099)